File: src/CovidWorkflow/CovidWorkflow/ModelsSqlServer/BedPatient.cs
using System;
using System.Collections.Generic;

namespace CovidWorkflow.ModelsSqlServer
{
    public partial class BedPatient
    {
        public long Idpatient { get; set; }
        public long Idbed { get; set; }
        public DateTime? DateModification { get; set; }

        public virtual Bed IdbedNavigation { get; set; }
        public virtual Patient IdpatientNavigation { get; set; }
    }
}

File: src/CovidAng/src/classes/BedPatient.ts
import { IdName } from './IdName';
import { BedWithRoom } from './BedWithRoom';

export class BedPatient {
  public idpatient: number;
  public idbed: number;
}
export class BedPatientWithName extends BedPatient{
  public idbedNavigation: BedWithRoom;
}